[PM_Feedbacks_M1.2 & M1.3] -- Fix alignment of label and remove "Awaiting Review" Label

Ash-2k3 opened the pull request:
## Overview

This PR addresses the feedbacks given in PM meeting. Overall, it removes "Awaiting Review" Label from opportunity-List-Item for translation-review dashboard, changes length-label's colour and text from "short" to "Short Translations" and aligns the length-label with list item.
https://docs.google.com/document/d/1TkZ8Ddhpi3aCdAZ5gCp2y0KCP9Z4zQZ6bA1rSoACDts/edit#bookmark=id.fcnf2cpeu86j
## Essential Checklist

- [ ] The **PR title** starts with "Fix #bugnum: " or "Fix part of #bugnum: ...",
followed by a short, clear summary of the changes.
- [x] I have followed the [instructions for making a code change](https://github.com/oppia/oppia/wiki/Make-a-pull-request).
- [x] The linter/Karma presubmit checks have passed on my local machine.
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [ ] I have assigned the correct reviewers to this PR (or will leave a
comment with the phrase "@{{reviewer_username}} PTAL" if I don't have
permissions to assign reviewers directly).

## Proof that changes are correct

## Before Changes

<img width="1121" alt="Screenshot 2023-08-06 at 12 57 23" src="https://github.com/oppia/oppia/assets/98253080/ab019d27-89ed-4b1d-9474-bae032863691">

## After Changes

<img width="1440" alt="Screenshot 2023-08-03 at 22 34 04" src="https://github.com/oppia/oppia/assets/98253080/1b8eb8d1-e71a-46cb-9be3-fa33f0b756ea">

#### Proof of changes on mobile phone

## Before Changes

<img width="1148" alt="Screenshot 2023-08-06 at 12 57 48" src="https://github.com/oppia/oppia/assets/98253080/e3a52cf1-89aa-4ae9-8230-ff148e74b7e8">

## After Changes

<img width="1440" alt="Screenshot 2023-08-03 at 22 34 14" src="https://github.com/oppia/oppia/assets/98253080/49d79a14-b27c-4c4f-9646-e8e2164b0ac7">

Ash-2k3 committed:
Change colour and fix alignment of length label and remove awaiting review from opportunity list

Ash-2k3 committed:
Fix border radius

Ash-2k3 committed:
Merge branch 'develop' into feedback_show_translation_size

Ash-2k3 commented:
@sagangwee PTAL, Thanks.

Ash-2k3 committed:
Fix failing tests

Ash-2k3 committed:
Fix lint

Ash-2k3 committed:
Update opportunities-list-item.component.spec.ts

sagangwee commented:
@Ash-2k3 I still see the "Awaiting Review" Label in the description screenshot?

Ash-2k3 commented:
> @Ash-2k3 I still see the "Awaiting Review" Label in the description screenshot?

I think you got confused, I have made length-label to look like "Awaiting Review" Label. And  I have added "Awaiting Review" as a heading aligned with "Back to Lessons".

sagangwee commented:
@Ash-2k3 Well the PR title says `remove "Awaiting Review" Label` and the description says `Overall, it removes "Awaiting Review" Label from opportunity-List-Item`, so that's why I was clarifying. I suggest making that more clear. But, I don't really understand the point of adding this label in the first place? Even the feedback doc you linked has a note: `Why do we need the “Awaiting review” label, if everything is “Awaiting review”? `. I have the same question. The label seems redundant?

Ash-2k3 committed:
Remove awaiting review label completely

Ash-2k3 commented:
> @Ash-2k3 Well the PR title says `remove "Awaiting Review" Label` and the description says `Overall, it removes "Awaiting Review" Label from opportunity-List-Item`, so that's why I was clarifying. I suggest making that more clear. But, I don't really understand the point of adding this label in the first place? Even the feedback doc you linked has a note: `Why do we need the “Awaiting review” label, if everything is “Awaiting review”? `. I have the same question. The label seems redundant?

I am not sure either, why this label was added for reviewers. It makes sense to have this label for translation submitters, as they can check whether it's still in review or has been accepted or rejected. 
I agree with you that "Awaiting Review" is reduntant both as a label or header for the dashboard. I have removed it and updated the screenshots too, PTAL, Thanks.

sagangwee commented:
@Ash-2k3 I'm still seeing the label in the screenshots?

Ash-2k3 commented:
> Member

Sorry for the confusion, Updated now! PTAL, Thanks

Ash-2k3 committed:
Merge branch 'develop' into feedback_show_translation_size

Ash-2k3 committed:
Merge branch 'develop' into feedback_show_translation_size

kapilag96 commented:
@Ash-2k3 can you please add the screenshots of how it looked like before and after your changes in your PR description?

Ash-2k3 commented:
@kapilag96, @sagangwee Done, PTAL.

Ash-2k3 committed:
Merge branch 'develop' into feedback_show_translation_size

kapilag96 commented:
So we'll never show awaiting review? If yes, where was it removed from? I don't see any such changes

Ash-2k3 commented:
> So we'll never show awaiting review? If yes, where was it removed from? I don't see any such changes

Please look at the screenshots carefully, in the before changes screenshots, you can see "Awaiting Review" in the opportunity list item and in after changes it has been removed.

gp201 commented:
Removing my assignment as I'm not a codeowner.

kapilag96 commented:
> > So we'll never show awaiting review? If yes, where was it removed from? I don't see any such changes
> 
> Please look at the screenshots carefully, in the before changes screenshots, you can see "Awaiting Review" in the opportunity list item and in after changes it has been removed.

@Ash-2k3  Right but from the changes on the PR, I only see one modified html file that changes `short` to `Short Translation`. There was a commit that removed a div for `Awaiting Review` but it was added in the same PR. So I was wondering what was going on there.

Ash-2k3 commented:
> > > So we'll never show awaiting review? If yes, where was it removed from? I don't see any such changes
> > 
> > 
> > Please look at the screenshots carefully, in the before changes screenshots, you can see "Awaiting Review" in the opportunity list item and in after changes it has been removed.
> 
> @Ash-2k3 Right but from the changes on the PR, I only see one modified html file that changes `short` to `Short Translation`. There was a commit that removed a div for `Awaiting Review` but it was added in the same PR. So I was wondering what was going on there.

Ah I see , well we don't need to remove the html tags for it since we are not removing it completely, "Awaiting Review" currently is shown for two cases, for translation reviewers and translation submitters. And it makes sense to have this label for translation submitters, as they can see whether the suggestion they submitter has been accepted/rejected or is still in review. 

We wanted it to remove it for translation reviewers, as it doesn't really make sense to have it. We just need to change the ngIf condition as I had done here https://github.com/oppia/oppia/pull/18719/files#diff-47ba742a44f0c0474af2678ca6743072341165b9e5544915b56234f7045b5a4bR127

sagangwee commented on core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.html:
```diff
@@ -98,13 +98,14 @@
-      <ng-container>
-          short
-      </ng-container>
+  <div class="translation-length-label-wrapper">
```
Does this class have any styling? If not, we can remove.

Ash-2k3 replied:
Done, Thanks.

sagangwee reviewed (approved):
@Ash-2k3 LGTM

Ash-2k3 committed:
Remove class

Ash-2k3 committed:
Merge branch 'develop' into feedback_show_translation_size

Ash-2k3 committed:
Merge branch 'develop' into feedback_show_translation_size
